### PR TITLE
Update Baseline Profile generation

### PR DESCRIPTION
**Pull request by keyboardsurfer**

### CODE REVIEW

File: benchmark/build.gradle.kts [hunk 1]
```diff
@@ -49,9 +48,21 @@ android {
...
     configureFlavors(this)
-    
+
     targetProjectPath = ":app"
     experimentalProperties["android.experimental.self-instrumenting"] = true
+
+    testOptions {
+        managedDevices {
+            devices {
+                maybeCreate<ManagedVirtualDevice>("pixel5Api32").apply {
```
Comment (alexvanyo) [inline benchmark/build.gradle.kts:58]: Nit: name is `pixel5Api32`, but `apiLevel` below is `30`.

**Review by alexvanyo — APPROVED** [review]

File: benchmark/build.gradle.kts [hunk 2]
```diff
@@ -49,9 +48,21 @@ android {
...
     configureFlavors(this)
-    
+
     targetProjectPath = ":app"
     experimentalProperties["android.experimental.self-instrumenting"] = true
+
+    testOptions {
+        managedDevices {
+            devices {
+                maybeCreate<ManagedVirtualDevice>("pixel5Api30").apply {
```
Comment (wojtek-kalicinski) [inline benchmark/build.gradle.kts:58]: why not just create<...>(..>) { 

}

Comment (keyboardsurfer) [reply benchmark/build.gradle.kts]: Updated.

File: benchmark/build.gradle.kts [hunk 3]
```diff
@@ -49,9 +48,21 @@ android {
...
     configureFlavors(this)
-    
+
     targetProjectPath = ":app"
     experimentalProperties["android.experimental.self-instrumenting"] = true
+
+    testOptions {
+        managedDevices {
+            devices {
+                create<ManagedVirtualDevice>("pixel5Api30").apply {
```
Comment (wojtek-kalicinski) [inline benchmark/build.gradle.kts:58]: `.apply{ }` not needed, just create<...>(...) { }

Comment (wojtek-kalicinski) [inline benchmark/build.gradle.kts:58]: .apply not needed, just create<>() {}

**Review by wojtek-kalicinski — APPROVED** [review]

**Pull request merged**